hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/EthereumSuite.java

```diff
@@ -1272,4 +1272,29 @@ final Stream<DynamicTest> callHtsSystemContractTest() {
                                     .hasKnownStatus(SUCCESS));
                 }));
     }
+
+    @HapiTest
+    final Stream<DynamicTest> negativeGasFailsGracefully() {
```

Comment:
This tests new code at `IngestChecker.assertThrottlingPreconditions`.  Needs a separate test for new code at `GasLimitDeterministicThrottle.allow`.  If there's a way to hapispec test throttles, don't know.

Reply:
The GasLimitDeterministicThrottle.allow txGasLimit<0 code path should never execute. I'll create a unit test for it but I don't think there's any way to create a hapi test.

Reply:
Unit test created